### ISSUE TITLE
[icetray] Remove ROOT version from I3TrayInfo (Trac #2057)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2049]
icetray project includes ROOT as a tool for the sole purpose of saving the version string in I3TrayInfo. TrayInfos are so rarely used and icerec is almost completely de-ROOT-ified. this is small, but I don't think anyone ever uses this information 

Migrated from https://code.icecube.wisc.edu/ticket/2057
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:38", 
    "description": "icetray project includes ROOT as a tool for the sole purpose of saving the version string in I3TrayInfo. TrayInfos are so rarely used and icerec is almost completely de-ROOT-ified. this is small, but I don't think anyone ever uses this information ", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067278746682", 
    "component": "combo core", 
    "summary": "[icetray] Remove ROOT version from I3TrayInfo", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-07-28T11:44:57", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180220T15:39:47: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 3031:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3031"
> Remove `root_version` from I3TrayInfo, closes #2049
> ```


* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:03:51: nwhitehorn commented:

> In changeset 168852:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168852"
> Remove `root_version` from I3TrayInfo, closes #2049
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:38: 
* olivas changed milestone from "September 2017" to ""
* olivas commented:

> Milestone September 2017 deleted
